tests/components/ipp/test_config_flow.py

```diff
@@ -364,17 +376,20 @@ async def test_full_zeroconf_tls_flow_implementation(
 
     assert result["step_id"] == "zeroconf_confirm"
     assert result["type"] == RESULT_TYPE_FORM
-    assert result["description_placeholders"] == {CONF_NAME: "EPSON123456"}
+    assert result["description_placeholders"] == {CONF_NAME: "EPSON XP-6000 Series"}
 
     result = await hass.config_entries.flow.async_configure(
         result["flow_id"], user_input={}
     )
 
     assert result["type"] == RESULT_TYPE_CREATE_ENTRY
-    assert result["title"] == "EPSON123456"
+    assert result["title"] == "EPSON XP-6000 Series"
 
     assert result["data"]
     assert result["data"][CONF_HOST] == "192.168.1.31"
-    assert result["data"][CONF_NAME] == "EPSON123456"
+    assert result["data"][CONF_NAME] == "EPSON XP-6000 Series"
     assert result["data"][CONF_UUID] == "cfe92100-67c4-11d4-a45f-f8d027761251"
     assert result["data"][CONF_SSL]
+
+    assert result["result"]
+    assert result["result"].unique_id == "cfe92100-67c4-11d4-a45f-f8d027761251"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5373293)